modules/drain/drain.c

```diff
@@ -5,7 +5,7 @@
 static bool is_draining = false;
 
 /*
- * Iff is_draining == true, prevents new allocations and denies allocation
+ * If is_draining == true, prevents new allocations and denies allocation
```

Comment:
`iff` means [if and only if](https://en.wikipedia.org/wiki/If_and_only_if), but the comment is clear enough either way.